src/clparser.cc

```diff
@@ -53,15 +53,15 @@ string CLParser::FilterShowIncludes(const string& line,
 
 // static
 bool CLParser::IsSystemInclude(string path) {
-  transform(path.begin(), path.end(), path.begin(), ::tolower);
+  transform(path.begin(), path.end(), path.begin(), [](const char c) { return static_cast<char>(::tolower(c)); });
```

Comment:
Ninja doesn't use any c++11 features and doesn't pass -std=c++11 to gcc/clang (where one has to explicitly opt in to that).